src/main/resources/application.properties

```diff
@@ -35,10 +35,17 @@ ccd.case.search.wildcards.allowed=false
 ccd.dm.domain=${CCD_DM_DOMAIN:https://api-gateway.dev.dm.reform.hmcts.net}
 
 auth.idam.client.baseUrl=${IDAM_USER_URL:http://localhost:4501}
-auth.provider.service.client.baseUrl=${IDAM_S2S_URL:http://localhost:4502}
-auth.provider.service.client.microservice=ccd_data
-auth.provider.service.client.key=${DATA_STORE_IDAM_KEY:AAAAAAAAAAAAAAAB}
-auth.provider.service.client.tokenTimeToLiveInSeconds=900
+
+# This property *must* be retained because it is required by the ServiceTokenParserConfiguration class in the
+# auth-checker-lib library, on which there is a dependency
+auth.provider.service.client.baseUrl=${IDAM_S2S_URL:http://rpe-service-auth-provider-saat.service.core-compute-saat.internal:4502}
+
+# Required for the ServiceAuthorisationApi class in service-auth-provider-java-client library
+idam.s2s-auth.totp_secret=${DATA_STORE_IDAM_KEY:AAAAAAAAAAAAAAAB}
+idam.s2s-auth.microservice=ccd_data
+# New CNP URL (locally, requires SSH tunnel and hosts file entry) - service-auth-provider-java-client library is *not*
```

Comment:
is saying new cnp url going to be very useful for long? seems to be more like something you should tell people in slack but don't need to add a comment here

Reply:
Good point - I might remove this